varia/sass/abstracts/_config-global.scss

```diff
@@ -50,59 +52,23 @@ $config-global: (
 			"xxl": normal,
 			"xxxl": normal,
 		),
-		/* Line Height */
+
 		"line-height": (
 			"base": strip-unit($typescale-base),
 			"body": 1.78,
 			"heading": 1.125,
 		),
 	),
 
-	/* Colors */
-	"color": (
-		"primary": (
-			"default": blue,
-			"hover": indigo,
-		),
-		"secondary": (
-			"default": red,
-			"hover": darkred,
-		),
-		"foreground": (
-			"default": #444444,
```

Comment:
I think this will break other child themes, not all of them are using CSS variables. A good example of one Varia child that doesn't is Maywood.

Reply:
To test how this change affects all the varia children you should rebuild all of their CSS, running `sh rebuild-child-themes.sh` from inside the varia theme folder.

Reply:
Right. Many child themes are breaking. I will go ahead and convert to CSS variables.
How do you suggest PRs. 
* include everything here
* create a single PR for all child themes
* or individual PR for each child theme

Reply:
I believe there's a reason why we didn't use CSS variables for all child themes but I can't remember why! I think both methods should live in varia and each child is using one or the other.